### ISSUE TITLE
Nix build failure

### ISSUE THREAD

**Issue by Ciflire** [Jas-SinghFSU/HyprPanel#655]
**Describe the bug**
Hyprpanel doesn't build on nix

**To Reproduce**
- first bad commit : a7b553725c1deb4a856ddd8582fa217451533ec3
- try to build with `nix build`

**Expected behavior**
I expect hyprpanel to build that way

**Desktop (please complete the following information):**
 - Distribution: NixOS
 - Window Manager/Desktop Environment: Hyprland

**Additional context**
Build failure logs
```
HyprPanel on  HEAD (a7b5537) [?] is 📦 v1.0.0 via 
➜ nix build
error: builder for '/nix/store/kyjb8d677y9k4yyvvh2xvywy2nh61hq9-hyprpanel.drv' failed with exit code 1;
       last 25 log lines:
       >
       > ✘ [ERROR] Could not resolve "astal/gtk3"
       >
       >     ../../../../build/hyqr9jkn9dq7xd5f5hjgshc0ws1zbsbv-source/src/components/menus/bluetooth/devices/device/DeviceIcon.tsx:2:20:
       >       2 │ import { Gtk } from 'astal/gtk3';
       >         ╵                     ~~~~~~~~~~~~
       >
       >   You can mark the path "astal/gtk3" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
       >
       > ✘ [ERROR] Could not resolve "astal/gtk3"
       >
       >     ../../../../build/hyqr9jkn9dq7xd5f5hjgshc0ws1zbsbv-source/src/components/settings/pages/theme/menus/network.tsx:5:20:
       >       5 │ import { Gtk } from 'astal/gtk3';
       >         ╵                     ~~~~~~~~~~~~
       >
       >   You can mark the path "astal/gtk3" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
       >
       > ✘ [ERROR] Could not resolve "astal/gtk3"
       >
       >     ../../../../build/hyqr9jkn9dq7xd5f5hjgshc0ws1zbsbv-source/src/components/settings/pages/theme/menus/media.tsx:5:20:
       >       5 │ import { Gtk } from 'astal/gtk3';
       >         ╵                     ~~~~~~~~~~~~
       >
       >   You can mark the path "astal/gtk3" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
       >
       For full logs, run 'nix log /nix/store/kyjb8d677y9k4yyvvh2xvywy2nh61hq9-hyprpanel.drv'.
```

**Comment by cmargiotta** [comment]
This is happening to me too

**Comment by benvonh** [comment]
it builds after a flake update, wait for https://github.com/Jas-SinghFSU/HyprPanel/pull/652

**Comment by benvonh** [comment]
lol now it segfaults instead

**Comment by Drazzy9295** [comment]
Just in case some other people are here looking for a quick fix, I managed to get it building by pinning to the last working version. The meson update (a7b553725c1deb4a856ddd8582fa217451533ec3) seemed to be the first bad commit.

Here's how it should look in your flake.nix:

```
{
  inputs = {
    # Hyprpanel Bar
    hyprpanel.url = "github:Jas-SinghFSU/HyprPanel/ba7e0dfd52172c9c63ee11da689dce49cf76ae1f"; # Pinned version
  };
}
```

**Comment by benvonh** [comment]
Good to close?

**Comment by Jas-SinghFSU** [comment]
Fixed via #669
